packages/@react-native-windows/cli/src/e2etest/__snapshots__/dependencyConfig.test.ts.snap

```diff
@@ -24,6 +24,34 @@ Object {
 }
 `;
 
+exports[`dependencyConfig - SimpleCSharpApp (Ignore react-native.config.js) 1`] = `
+Object {
+  "folder": "D:\\\\Source\\\\react-native-windows\\\\packages\\\\@react-native-windows\\\\cli\\\\src\\\\e2etest\\\\projects\\\\SimpleCSharpApp",
```

Comment:
thanks for adding tests :) I think you'll want these to be relative paths so they work everywhere

Reply:
Thanks! Definitely didn't intend to have hard-coded paths here 😛